src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -0,0 +1,112 @@
+<template id="limited-restoration-panel">
+  <v-radio-group class="mt-0 pt-0 ml-8" v-model="selectMonths" mandatory>
+    <v-radio id="twentyFour-radio" name="selectMonths" label="Two Years" :value=24 />
+    <v-radio id="eighteen-radio" name="selectMonths" label="18 months" :value=18 />
+    <v-radio id="twelve-radio" name="selectMonths" label="12 months" :value=12 />
+    <v-radio id="six-radio" name="selectMonths" label="6 months" :value=6 />
+    <v-row class="ml-0 mt-0">
+      <v-radio id="custom-months" name="selectMonths" value="customMonths" >
+      </v-radio>
+      <v-text-field
+      class="shrink"
+      hide-details="auto"
+      type="number"
+      dense
+      hide-spin-buttons
+      min=1
+      max=24
+      :rules="monthRules"
+      v-model="numberOfMonths"
+      :disabled="!customMonths"
+      @change="onMonthsChanged"
+      filled />
+      <div class="ml-2 mt-2 text--secondary">month(s)</div>
```

Comment:
This assumes that Vuetify theme colour "secondary" is assigned to the desired gray. I think it would be better t specify one of our specified colours (gray7 or 9 I think).